### PR TITLE
Other services to protobuf

### PR DESCRIPTION
**Pull request by dvush**
## Overview

---

See: https://jira.bf.local/browse/ECR-XYZW

### Definition of Done

- [x] There are no TODOs left in the merged code
- [ ] Change is covered by automated tests
- [ ] Benchmark results are attached (if applicable)
- [x] The [coding guidelines] are followed
- [ ] Public API has proper documentation
- [x] Changelog is updated if needed (in case of notable or breaking changes)
- [x] The continuous integration build passes

[coding guidelines]: https://github.com/exonum/exonum/blob/master/CONTRIBUTING.md#conventions

### CODE REVIEW

File: examples/cryptocurrency/Cargo.toml [hunk 1]
```diff
@@ -18,12 +18,18 @@ circle-ci = { repository = "exonum/exonum" }
 
 [dependencies]
 exonum = { version = "0.9.0", path = "../../exonum" }
+exonum_derive = { version = "0.9.0", path = "../../exonum_derive" }
 failure = "0.1.2"
 serde = "1.0.0"
 serde_derive = "1.0.0"
 serde_json = "1.0.0"
+protobuf = "=2.2.0"
```
Comment (vitvakatu) [inline examples/cryptocurrency/Cargo.toml]: Is strict version number necessary?

Comment (aleksuss) [reply examples/cryptocurrency/Cargo.toml]: I suppose not

Comment (vitvakatu) [reply examples/cryptocurrency/Cargo.toml]: So I think we should remove strict versions if it's possible

**Comment by aleksuss** [comment]
@dvush don't forget about CHANGELOG please

File: examples/cryptocurrency/Cargo.toml (hunk 1 above)
Comment (aleksuss) [reply examples/cryptocurrency/Cargo.toml]: I guess it's actual for crates with version less then 1.0

**Comment by dvush** [comment]
@vitvakatu 
If you substract .proto files than changes would be small.
And it is not temporary, at least I do not think so, since you have to have .proto files to have interaction between service and client.

File: examples/cryptocurrency/Cargo.toml (hunk 1 above)
Comment (dvush) [reply examples/cryptocurrency/Cargo.toml]: Ok, I'll remove it in the batch, not in this PR.

**Pull request merged**